tests/server.py

```diff
@@ -303,6 +306,18 @@ def connectTCP(self, host, port, factory, timeout=30, bindAddress=None):
 
         return conn
 
+    def advance(self, amount):
+        # run any "callFromThread" callbacks before anything registered with
+        # "callLater"
```

Comment:
What do you mean by "before anything registered with "callLater"?

Reply:
well, `super().advance()` runs callbacks that were registered with `callLater`. Perhaps I'll reword.